app/views/layout.erb

```diff
@@ -6,8 +6,6 @@
     <title>Tuition Payment System</title>
   </head>
   <body>
-
-    <%= yield %>
-
+    <main><%= yield %></main>
```

Comment:
Makes more sense to have `<main>` in layout so we don't have to add it in every template.